File: Models/Developer.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace GameShopWebApi.Models{
    
    public class Developer{
        [Key]
        public int ID {get; set;}
        public string Naziv {get; set;}
        public int GodinaZasnivanja {get; set;}
        public int BrojIgrica {get; set;}
        public int BrojIgricaNaStanju { get; set;}
    }
}

File: Klijent/developer.js
export class Developer {
    constructor(id, naziv, godinaZasnivanja, brojIgrica){
        this.id = id;
        this.naziv = naziv;
        this.godinaZasnivanja = godinaZasnivanja;
        this.brojIgrica = brojIgrica;
        this.brojIgricaNaStanju = 0;
    }

    updateDeveloper(plusminus){
        if(plusminus == 1){
            this.brojIgricaNaStanju++;
        }
        else{
            this.brojIgricaNaStanju--;
        }
    }

}